src/updater.rs

```diff
@@ -269,70 +269,68 @@ pub fn update_batch(
     debug!("getting bulk profiles");
     let mut rt = Runtime::new()?;
     let profiles_iter = cis_client.get_users_iter(None)?;
-    for profiles in profiles_iter {
-        if let Ok(profiles) = profiles {
-            info!("{}", profiles.len());
-            rt.block_on(
-                async move {
+    for profiles in profiles_iter.flatten() {
+        info!("{}", profiles.len());
```

Comment:
I just changed the indenting here after removing the `if let Ok(profiles) = profiles` and changes the iter to `flatten()` above.